posts/2019-04-08-roadmap.md

```diff
@@ -0,0 +1,128 @@
+[survey]: https://blog.rust-lang.org/2018/11/27/Rust-survey-2018.html
+[blog-2019]: https://blog.rust-lang.org/2018/12/06/call-for-rust-2019-roadmap-blogposts.html
+[read-rust]: https://readrust.net/rust-2019/
+[rfc]: https://github.com/rust-lang/rfcs/blob/master/text/2657-roadmap-2019.md
+
+In short, 2019 will be a year of *rejuvenation* and *maturation* for the Rust
+project. Much of the focus is on strengthening our foundations and paying
+down debt, both technical and organizational.
+
+2018 was a fantastic year for production usage of Rust. Some of the largest
+names in tech have started relying on Rust as a key part of their stack.
+Smaller companies have started to build great things with our technology as
+well.
+
+Shipping the 2018 edition was both a herculean task and a great success. But
+it also created a lot of debt, both technical and organizational. When
+reading the Rust 2019 blog posts, and when having conversations with team
+members at the Rust All Hands, a general theme developed: 2019 should still
+be a year of shipping, but a certain kind of shipping. Words like confidence,
+maturity, practicality, productivity, sustainability, and stability were
+often used. This should be a year of reflection, one of polish, one of
+finishing plans that were started long ago.
+
+In some ways, it's easier to describe what this year should not be, rather
+than what it should be. This should not be a year of dreaming up large new
+features, one of drastic change, or one that makes your old code feel
+obsolete.
+
+With this general theme in mind, here are the high-level plans from each of
+the teams for the upcoming year, in alphabetical order. The summaries are
+meant to show plans in broad strokes and how they fit together for the
+project.
+
+[The Cargo team plans to](https://github.com/rust-lang/rfcs/blob/master/text/2657-roadmap-2019.md#cargo):
+
+* Better support cross-compilation
+* Better support plugins, aka "custom cargo commands"
+* Finish up some "almost done" features like custom registries
+
+
+[The community team plans to](https://github.com/rust-lang/rfcs/blob/master/text/2657-roadmap-2019.md#community):
+
+* Invest in projects like RustBridge, event support, the YouTube channel and
+  modernizing the community calendar
+* Work on internationalization
+* Help the teams communicate better with the wider community
+
+[The compiler team plans to](https://github.com/rust-lang/rfcs/blob/master/text/2657-roadmap-2019.md#compiler):
+
+* Continue to reduce compile times, as much as possible
+* Improve IDE integrations
+* Improve compiler-internal documentation and organization
+
+[The crates.io team plans to](https://github.com/rust-lang/rfcs/blob/master/text/2657-roadmap-2019.md#cratesio):
+
+* Grow the team
+* Pay down techincal debt
+
+[The dev tools team plans to](https://github.com/rust-lang/rfcs/blob/master/text/2657-roadmap-2019.md#dev-tools):
+
+* Be a bit of a "commons" space for cross-cutting tools concerns
```

Comment:
Prob should include the "some individual subteams have their own roadmap, see [rustdoc] and [IDEs]" bit, since that's the actual meat of the roadmap there.